src/utils/payments/pawapay/pawapay.ts

```
@@ -0,0 +1,473 @@
+    });
+
+    if (!response.ok) {
+      throw new Error(`Failed to check deposit status: ${response.statusText}`);
+    }
+
+    return response.json();
+  } catch (error) {
+    console.error('Error checking deposit status:', error);
+    throw error;
+  }
+}
```

Biome (1.9.4)

[error] Duplicate class member name "checkDepositStatus"



(lint/suspicious/noDuplicateClassMembers)